bundles/org.openhab.binding.mqtt/src/main/resources/OH-INF/i18n/mqtt.properties

```diff
@@ -98,6 +98,8 @@ actionInputRetainLabel = Retain
 actionInputRetainDesc = Retain message
 actionLabel = publish an MQTT message
 actionDesc = Publishes a value to the given MQTT topic.
+actionRetainLabel = publish an MQTT message with retain option
```

Comment:
I'd say "with given retain option".

Otherwise it sounds like this will always send retain: true

Reply:
Thanks for the correction